doc/source/user_guide/computation.rst

```diff
@@ -884,9 +884,10 @@ and a ratio of :math:`1 - \alpha` we have
     &= \alpha x_t + (1 - \alpha)[x_{t-1} + (1 - \alpha) x_{t-2} + ...]\alpha\\
     &= \alpha x_t + (1 - \alpha) y_{t-1}
 
-which shows the equivalence of the above two variants for infinite series.
-When ``adjust=True`` we have :math:`y_0 = x_0` and from the last
-representation above we have :math:`y_t = \alpha x_t + (1 - \alpha) y_{t-1}`,
+which is the same expression as ``adjust=False`` above and therefore
+shows the equivalence of the two variants for infinite series.
+When ``adjust=False`` we have :math:`y_0 = x_0` and
+:math:`y_t = \alpha x_t + (1 - \alpha) y_{t-1}`,
```

Comment:
This is a comma splice.  Put a period here, and then capitalize "therefore" (also add a comma after it).

Reply:
Done. Agreed on comma splice. I don’t think the additional commas are strictly necessary but have added them anyway.

Reply:
Looks good.  Thanks!